clang/test/FixIt/format.cpp

```diff
@@ -1,5 +1,5 @@
-// RUN: %clang_cc1 -fsyntax-only -verify -Wformat %s
-// RUN: %clang_cc1 -fsyntax-only -fdiagnostics-parseable-fixits -Wformat %s 2>&1 | FileCheck %s
+// RUN: %clang_cc1 -fsyntax-only -verify -Wformat-pedantic %s
```

Comment:
We're missing a comparison between `-Wformat` and `-Wformat-pedantic` in terms of handling scoped enumerations. We should have a pair of RUN lines to show that this diagnostic is in pedantic only.

One thing that helps with testing this is to use something like `-verify=expected,pedantic` and `// pedantic-warning {{msg}}`: https://clang.llvm.org/docs/InternalsManual.html#verifying-diagnostics